### PR TITLE
bump to 1.11.0-beta-1 (for review only, do not merge)

### PR DESCRIPTION
**Pull request by egieseke**
# 1.11.0-beta-1
- ABI Interaction Support for JAVA SDK (#268)
- Bug fix for `logs` on `PendingTransactionResponse` (#275)
- Add WaitForConfirmation function (#274)
- Better error message on encoding exception. (#258)
- Revert "Revert "Fix ABI source code position for ABI feature (#260)""
- Revert "Revert "Add ABI encoding support (#255)""

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 1.11.0-beta-1
+- Bump bcprov-jdk15to18 to 1.69 (#267)
+- ABI Interaction Support for JAVA SDK (#268)
+- Add keyreg txn encoding tests (#269)
+- Stop empty state proof key from being encoded in txn (#270)
+- Add WaitForConfirmation function (#274)
+- Bug fix for logs on PendingTransactionResponse (#275)
```
Comment (jasonpaulos) [inline CHANGELOG.md:7]: PR #267 was never merged and PRs #269 and #270 did not target develop, so none of these should be in the changelog

Comment (egieseke) [reply CHANGELOG.md]: Removed those PRs from the changelog and updated the PR description.

Comment (egieseke) [reply CHANGELOG.md]: Regenerated PR list.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# 1.11.0-beta-1
+- ABI Interaction Support for JAVA SDK (#268)
+- Bug fix for `logs` on `PendingTransactionResponse` (#275)
+- Add WaitForConfirmation function (#274)
+- Better error message on encoding exception. (#258)
+- Revert "Revert "Fix ABI source code position for ABI feature (#260)""
+- Revert "Revert "Add ABI encoding support (#255)""
```
Comment (jasonpaulos) [inline CHANGELOG.md:7]: nit: could you remove the revert language from these? E.g. `Fix ABI source code position for ABI feature (#260)` is a good description (we reverted & un-reverted some commits around the previous release)

**Review by jasonpaulos — APPROVED** [review]
I left a suggestion for a cleaner changelog, but otherwise looks good

**Review by algojack — APPROVED** [review]
lgtm

**Pull request merged**